lib/carto_metadata.js

```diff
@@ -599,5 +599,63 @@ module.exports = function(redis_opts) {
         });
     }
 
+    me.getMultipleUserParams = function (username, fields, callback) {
+        const redisKey = this.user_key({ username: username });
+
+        this.mRetrieve(this.user_metadata_db, redisKey, fields, function (err, dbValues) {
+            if (err) {
+                return callback(err, null);
+            } else {
+                const element = {};
+                let found = false;
+
+                fields.forEach((field, pos) => {
+                    element[field] = dbValues[pos];
+                    found = found || !!dbValues[pos];
+                });
+
+                if (found) {
+                    return callback(null, element);
+                } else {
+                    return callback(null, null);
+                }
+            }
+        });
+    };
+
+    me.getUserBasicInfo = function (username, callback) {
+        const fields = ['id', 'database_host', 'database_name'];
+        this.getMultipleUserParams(username, fields, callback);
+    };
+
+    const apiKeyFields = [
+        'user',
+        'type',
+        'dbRole',
```

Comment:
let's keep snake-case for keys stored in Redis and camel-case for keys to return to the client.